tests/DevApps/B2CWebAppCallsWebApi/Client/appsettings.json

```diff
@@ -16,8 +16,8 @@
             "https://fabrikamb2c.onmicrosoft.com/tasks/read",
             "https://fabrikamb2c.onmicrosoft.com/tasks/write"
         ],
-        "BaseUrl": "https://localhost:44332",
-        "UserFlow": "b2c_1_susi"
+        "BaseUrl": "http://localhost:44332"
+        //"UserFlow": "b2c_1_susipolicy"
```

Comment:
why comment it out?

Reply:
Good catch it was left over from troubleshooting, uncommented it now.